test/models/lib/user_parameter_sanitiser_test.rb

```diff
@@ -24,31 +24,6 @@ class UserParameterSanitiserTest < ActiveSupport::TestCase
     end
   end
 
-  context "with ActionController params object" do
```

Comment:
How come these tests have been removed?

Reply:
Mentioned in the commit message which removes them: https://github.com/alphagov/signon/pull/589/commits/a4212964fa9c83793a2281956892b1670dabb83c